ui/src/main/resources/CKEditor/MacroService.xml

```diff
@@ -181,7 +181,7 @@
       'deprecated': $parameterDescriptor.deprecated,
       'advanced': $parameterDescriptor.advanced,
       'defaultValue': $parameterDescriptor.defaultValue,
-      'type': $parameterDescriptor.parameterType,
+      'type': $parameterDescriptor.displayType,
```

Comment:
What about the wiki-based rendering macros? Can they specify the display type? If not, then does the display type fall back on the parameter type in their case? I want to make sure the wiki-based rendering macros can still get a picker easily.

Reply:
Answering myself: yes, it does fallback on the ``parametertType`` for wiki-based rendering macros, but there's another issue: ``displayType`` is only available since XWiki 11.0 but CKEditor needs to work with older versions so ``MacroService`` needs to fall back on ``parametertType`` also.

Reply:
Sorry for not answering, indeed there was no fallback.